### ISSUE TITLE
Support styles

### ISSUE THREAD

**Issue by oysterlovers** [ilian6806/stable-diffusion-webui-state#6]
<img width="922" alt="2023-03-23_21-17-34_%pn" src="https://user-images.githubusercontent.com/107333049/227225053-f295e000-4a84-49ff-ad43-c7b14d11cf3f.png">

**Comment by ilian6806** [comment]
Yes, it`s much much complicated with multiselect dropdowns in gradio, but I give it a try :)

**Comment by ilian6806** [comment]
Ok, I applied some dark JS magic and it seems to be working now :) Haven't tested it much but however...

**Comment by leppie** [comment]
Not working :( 

`gradioApp().querySelector(`#${id}.multiselect`);` seems to be a better choice. I dont even have nodes with those CSS classes you used. This works for saving at least.

But restoring the options, it fails to find `select.parentNode.querySelectorAll('ul li')` as it is not created (yet?).

I can see what you trying to do (it is a lot more than I would have tried!), but it seems quite fragile :(

**Comment by leppie** [comment]
Found hacky way to get it working 

```
let option = value.pop();
state.utils.triggerMouseEvent(input, 'focus'); 
```

The click event does not seem to trigger the popup.

**Comment by leppie** [comment]
Another issue, it seems to add the items in reverse. 

This seems to do the trick:

```
value = value.split(',').reverse();
```

**Comment by ilian6806** [comment]
FIrst of all, thank you a lot for your feedback :)
I updated my SD version to latest and there were A LOT of changes there.
- gradio version was updated
- selects is not native html elements anymore, but custom ones
- get_uiCUrrentTab method does not work anymore
- onUiTabChange callback not working properly at the moment
- changed css classes almost everywhere

So yes, as this is almost entirely JS extension, it depends on some html structure when there is no id attributes there and it is very fragile and hard to support.

Today I fixed most of this issues and found some new ones... I also applied your fixes. You can update the extension and check if its working for you. If not, please check js console for errors, to give me a hint for eventual problems.

Cheers :)

**Comment by leppie** [comment]
Updated, styles seem fine. single selects appear to be working fine too.
